src/rotating_caliper.py

```diff
@@ -74,22 +76,22 @@ def get_angle_of_corner(self, corner:int):
         return angle_between_vectors(Vector2d(self.corners[(corner-1)%4], self.corners[corner]), Vector2d(self.corners[(corner+1)%4], self.corners[corner]))
 
 
-    def get_scaled_geometry(self, scale_to:float=1, prism_z_dim:float=1) -> Geometry:
+    def get_scaled_geometry(self, scale_to:float=1, prism_z_dim_mm:float=1) -> Geometry:
         vec1 = Vector2d(self.corners[1], self.corners[0])
         vec2= Vector2d(self.corners[3], self.corners[0])
         x_rectangle = vec1.get_norm2()
         y_rectangle = vec2.get_norm2()
         theta = angle_between_vectors(Vector2d([1,0], [0,0]), vec1)
-        return Geometry(center= Pose(x=self.center[0]*scale_to,
-                                    y = self.center[1]*scale_to,  
+        return Geometry(center= Pose(x=self.center[0]*scale_to * TO_MM,
+                                    y = self.center[1]*scale_to * TO_MM,  
                                     z= 0, 
                                     o_x = 0, 
                                     o_y = 0, 
                                     o_z = 1,
                                     theta = theta),
-                                    box = RectangularPrism(dims_mm=Vector3(x = x_rectangle*scale_to,
-                                                                   y = y_rectangle*scale_to,
-                                                                   z = prism_z_dim)))
+                                    box = RectangularPrism(dims_mm=Vector3(x = x_rectangle*scale_to * TO_MM,
```

Comment:
[opt] I might instead add this to the detector.py where you send the scale_to. Alternatively creating a scalar variable at the top of this function which is:

`scalar = scale_to * TO_MM`

this would be consistent with the `prisim_z_dim_mm` which is in millimeters. If you wanted you could also rename to `scale_to_mm`

Reply:
done